tests/tools/microServer/uWServer.py

```diff
@@ -36,7 +36,7 @@
 import time
 test_mode_enabled = True
 lookup_key_ = "{PATH}"
-__version__ = "1.0"
+__version__ = "1.1"
```

Comment:
We are adding a new key to the session definition, so I thought it would be good to bump the minor version.

Are there any other consequences of doing this?

Reply:
Following up: This is the version of the command-line tool that is printed and not a schema version. So it is safe to change it.